psd-web/app/helpers/investigations/ts_instigations_helper.rb

```diff
@@ -0,0 +1,67 @@
+module Investigations::TsInstigationsHelper
```

Comment:
```suggestion
module Investigations::TsInvestigationsHelper
```
(also rename the file).